### ISSUE TITLE
Add new contributors article

### ISSUE THREAD

**Issue by ShirazSuleman** [freeCodeCamp/how-to-contribute-to-open-source#94]
Hi, I found a really useful guide for new open source contributors (https://github.com/collections/choosing-projects) and thought it would fit under the "Useful articles for new open source contributors" section.

If you guys agree, I'd like to add it :)

**Comment by Puugu** [comment]
I thought it looked really helpful, so I went ahead and added it (getting points for Hacktoberfest!!) Thanks for the article. It was a good read.

**Comment by ShirazSuleman** [comment]
That sucks man, it is literally what I wanted to do hence me mentioning it in the initial issue.

**Comment by Puugu** [comment]
My bad, Shiraz. I totally misunderstood. I thought if you wanted to add it, you would have added it in your own pull request. I thought you were trying to help out other people who were trying to learn pull requests and offered it as a suggestion.

Were you just submitting it to contribute, or were you trying to accomplish some sort of goal? I'd love to try to help you find something to make it up to you.

**Comment by ShirazSuleman** [comment]
I wanted to add it but am new to contributing and was told to create an issue requesting the change to get confirmation that I can go ahead. I'm also trying to get pull requests for Hacktoberfest.

**Comment by Puugu** [comment]
Well, I found this article:
https://opensource.guide/how-to-contribute/
That I was going to contribute, but I feel like I owe you one, so I'll leave it in your hands.

I'd also recommend this repository for Hacktoberfest:
https://github.com/AliceWonderland/hacktoberfest
You can do each of the challenges as a separate pull request if you want.
I also found this one to be helpful for explaining the process (it was the first guide where I understood what was going on):
https://github.com/Roshanjossey/first-contributions

**Comment by ShirazSuleman** [comment]
Thanks but that article is already in the README.md. Will take a look at the other links though.

**Comment by erictleung** [comment]
> Hi, I found a really useful guide for new open source contributors (https://github.com/collections/choosing-projects) and thought it would fit under the "Useful articles for new open source contributors" section.

@ShirazSuleman thanks for suggesting this article! This was already suggested in a PR made before you created this issue https://github.com/freeCodeCamp/how-to-contribute-to-open-source/pull/90. I'm waiting on them to format it. If they don't work on it by next week, then you or someone else can take it over.
